src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs

```diff
@@ -209,6 +199,38 @@ public string DeploymentButtonLabel
             }
         }
 
+        public bool Expanded
+        {
+            get => _expanded;
+
+            set
+            {
+                _expanded = value;
+                
+                if (value == true)
```

Comment:
simply `if (value)` is preferred

Reply:
sure.  The main issue is that (boolvar == true) is redundant, just (boolvar) suffices

Reply:
If you feel you need to write (boolvar == true) for readability, that suggests that boolvar could be better named.  In your example, `_expanded` could be `_isExpanded`.  Either is fine .